html/interactive-guides/microprofile-config/download-from-injection.html

```diff
@@ -14,5 +14,5 @@
 <head>
 </head>
 <body>
-    <p style="color: red;">Downloaded music from ftp://music.com/us/download.</p>
+    <p style="color: darkgreen;">Downloaded music from ftp://music.com/us/download.</p>
```

Comment:
I want each html to have different color with the download message. Darkgreen is already taken. Either change back to red or change to a different color than darkgreen and slateblue.